python/lsst/sims/catUtils/mixins/PhotometryMixin.py

```diff
@@ -705,13 +721,56 @@ def get_lsst_magnitudes(self):
         """
         getter for LSST magnitudes of solar system objects
         """
-
         if not hasattr(self, 'lsstBandpassDict'):
             self.lsstBandpassDict = BandpassDict.loadTotalBandpassesFromFiles()
 
         return self._magnitudeGetter(self.lsstBandpassDict, self.get_lsst_magnitudes._colnames)
 
 
+    def get_magFilter(self):
+        """
+        Generate the magnitude in the filter of the observation.
+        """
+        if not hasattr(self, 'lsstBandpassDict'):
+            self.lsstBandpassDict = BandpassDict.loadTotalBandpassesFromFiles()
+        mags = self._magnitudeGetter(self.lsstBandpassDict, self.lsstBandpassDict.keys())
+        mag_keys = self.lsstBandpassDict.keys()
+        magFilter = mags[mag_keys.index(self.obs_metadata.bandpass)]
+        return magFilter
+
+    def get_SNR(self):
```

Comment:
should this method be renamed get_magSNR?

I can imagine adding a compound getter get_SNR that gets SNR for all of the bandpasses in the near future.